src/main/startgg.ts

```diff
@@ -815,6 +816,7 @@ function gqlParticipantToParticipant(participant: ApiParticipant): Participant {
     displayName: participant.gamerTag,
     pronouns: participant.player.user?.genderPronoun || '',
     prefix: participant.prefix || '',
+    id: participant.id || '',
```

Comment:
I assume this `id` is the same as `participantId`, but haven't confirmed this / run this function